src/main/java/Control/Connect/DbProvider.java

```diff
@@ -105,13 +109,13 @@ public ResultSet getRs() {
     private void afterSqlExecution(boolean successful) {
         try {
             // when query was unsuccesfull do not commit
-            if (successful == false) {
+            if (successful == false && isItTransaction) {
```

Comment:
k comu je toto isItTransaction?

Reply:
aby sme nerobili rollback v pripade ze sa nejednalo o transakciu chybnu ale iba o gety, lebo tak to je useless